packages/ui-components/src/components/Table/TablePaginator.js

```diff
@@ -61,6 +71,10 @@ export const TablePaginator = React.memo(
       [onChangeRowsPerPage],
     );
 
+    if (count <= rowsPerPage) {
+      return null;
+    }
```

Comment:
I think I'm missing something here. Does moving this check after creating the hooks achieve something?

Reply:
yeah it just ensures that the hooks are not called conditionally which is one of the rules of hooks. It can cause weird bugs with how they store state internally if you do. It was just bought to my attention by the updated linting rules that Kosas added